src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -10,4 +10,14 @@ export default interface ClientMetricReport {
    * Gets raw client media metrics
    */
   getObservableMetrics(): { [id: string]: number };
+
+  /**
+   * Gets video upstream client media metrics
+   */
+  getObservableVideoUpstreamMetrics(): { [id: string]: {} };
```

Comment:
Can we just have one single VideoMetrics? Dont think we need to seperate them out

Reply:
We can. Sure, I can combine them into one.